docs/config/README.md

```diff
@@ -256,23 +256,73 @@ MINIO_ETCD_COMMENT          (sentence)  optionally add a comment to this setting
 
 This behavior is consistent across all keys, each key self documents itself with valid examples.
 
-## Environment only settings (not in config)
+## Dynamic systems without restarting server
 
-#### Usage crawler
-> NOTE: Data usage crawler is not supported under Gateway deployments.
+Following sub-systems are dynamic i.e configuration parameters for each sub-systems can be changed while the server is running without any restarts.
+
+```
+api                   manage global HTTP API call specific features, such as throttling, authentication types, etc.
+heal                  manage object healing frequency and bitrot verification checks
+crawler               manage crawling for usage calculation, lifecycle, healing and more
+```
 
-Data usage crawler is enabled by default, following ENVs allow for more staggered delay in terms of usage calculation.
+> NOTE: if you set any of the following sub-system configuration using ENVs, dynamic behavior is not supported.
 
-The crawler adapts to the system speed and completely pauses when the system is under load. It is possible to adjust the speed of the crawler and thereby the latency of updates being reflected. The delays between each operation of the crawl can be adjusted by the `MINIO_DISK_USAGE_CRAWL_DELAY` environment variable. By default the value is `10`. This means the crawler will sleep *10x* the time each operation takes.
+### Usage crawler
 
-This will in most setups make the crawler slow enough to not impact overall system performance. Setting `MINIO_DISK_USAGE_CRAWL_DELAY` to a *lower* value will make the crawler faster and setting it to 0 will make the crawler run at full speed (not recommended). Setting it to a higher value will make the crawler slower, further consume less resources.
+Data usage crawler is enabled by default, following config settings allow for more staggered delay in terms of usage calculation. The crawler adapts to the system speed and completely pauses when the system is under load. It is possible to adjust the speed of the crawler and thereby the latency of updates being reflected. The delays between each operation of the crawl can be adjusted by the `mc admin config set alias/ delay=15.0`. By default the value is `10.0`. This means the crawler will sleep *10x* the time each operation takes.
+
+This will in most setups keeps the crawler slow enough to not impact overall system performance. Setting `delay` key to a *lower* value will make the crawler faster and setting it to 0 will make the crawler run at full speed (not recommended in production). Setting it to a higher value will make the crawler slower, further consume lesser resources.
```

Comment:
```suggestion
In most setups this will keep the crawler slow enough to not impact overall system performance. Setting the `delay` key to a *lower* value will make the crawler faster and setting it to 0 will make the crawler run at full speed (not recommended in production). Setting it to a higher value will make the crawler slower, consuming less resources with the trade off of not collecting metrics for operations like healing and disk usage as fast.
```